python/py-scipy/Portfile

```diff
@@ -40,6 +39,15 @@ compilers.setup         require_fortran -clang -gcc44 -gcc45 -gcc46 \
 github.livecheck.regex {([0-9.-]+)}
 
 if {${name} ne ${subport}} {
+    if {${python.version} < 312} {
+        python.pep517        no
+    } else {
+        depends_build-append \
+                    port:py${python.version}-meson-python \
```

Comment:
This will be added automatically if you set `python.pep517_backend meson`, which you should do, because otherwise you get an unneeded dependency on setuptools instead.

Reply:
Thanks! Turns out I did not need the cmake dependency either, so I removed both and replaced them with `python.pep517_backend meson` as suggested.